.ci/deploy/devfile-registry.yaml

```diff
@@ -148,13 +171,20 @@ objects:
             enabled: true
             path: /metrics
 
+    devfile-registry-hosts.json: |
+      {
+        "Community": {
+          "url": "http://localhost:8080"
+        }
+      }
+
 parameters:
 - name: DEVFILE_INDEX_IMAGE
-  value: quay.io/devfile/devfile-index
+  value: quay.io/jcollier/devfile-index
```

Comment:
Is there any reason why we use the personal image instead of the one under devfile?

Reply:
No, that was just there for testing and accidentally got left in

Reply:
Fixed, thanks for catching that!